README.md

```diff
@@ -66,121 +71,83 @@ This approach dramatically reduces latency and reduces load on the OP's servers.
 
 ## Overview
 
-This specifications introduces the following mechanisms to provide VCs and VPs to RPs:
+Verifiable Credentials and Verifiable Presentations can be added to a OpenID Connect UserInfo response or an ID Token.
```

Comment:
I don't fully get why we need the following 3 sentences since the real definition is given in "ID Token Extensions".

I suggest to remove the additional heading "ID Token Extensions" and move (a simplified version of) the first 3 sentences to the end of the section. 

Text could be: 

"Those claims can be added to ID Tokens, Userinfo responses as well as Access Tokens and Introspection response. "